### PR TITLE
Feature/create operation service tests

### PR DESCRIPTION
**Pull request by Maksim-Trolina**

### CODE REVIEW

File: Tests/BrokerServiceTests/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void Test1()
+        {
+            Assert.Pass();
```
Comment (nmamro) [inline Tests/BrokerServiceTests/UnitTest1.cs]: The same situation as in your another PR

File: Tests/OperationServiceTests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void Test1()
+        {
+            Assert.Pass();
```
Comment (nmamro) [inline Tests/OperationServiceTests/UnitTest1.cs:15]: And her

File: Tests/BrokerServiceTests/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace BrokerServiceTests
```
Comment (nmamro) [inline Tests/BrokerServiceTests/UnitTest1.cs]: Strange to see this in _create operation service tests_

**Review by nmamro — DISMISSED** [review]

File: Tests/BrokerServiceTests/UnitTest1.cs (hunk 3 above)
Comment (Maksim-Trolina) [reply Tests/BrokerServiceTests/UnitTest1.cs]: > Strange to see this in _create operation service tests_

it was added back in the previous request, in this I didn’t add it specifically

**Pull request merged**